### ISSUE TITLE
Error: Cannot find module '../../lib/util'

### ISSUE THREAD

**Issue by 4everinbeta** [krakenjs/generator-swaggerize#85]
I am trying the quickstart example from the readme and am getting the following error:

```
$ yo swaggerize
module.js:327
    throw err;
    ^

Error: Cannot find module '../../lib/util'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
```

I completely admit ignorance but am usually able to find answers to my problems. This one seems to have me at a loss.

**Comment by subeeshcbabu** [comment]
Can you please share following details:

Platform (mac, windows etc ), Node version, NPM version, `yo`, and `generator-swaggerize` version

**Comment by subeeshcbabu** [comment]
Looks like you picked up a pre-release publish and somehow the published module is missing directories. Can you re-install and try again?

**Comment by subeeshcbabu** [comment]
`npm publish` somehow ignores the `lib` directory. trying to find out why. `.npmignore`seems to be OK.

**Comment by subeeshcbabu** [comment]
For the pre-release 3.0.0-alpha, `lib` was missing in the `files` field of `package.json`. (Fixed the issue in `3.0.0-alpha.4`). Also the `latest` tag should be now pointing to stable version `2.0.2`. 
If you want to try `alpha` release for the next Major version, use `3.0.0-alpha.4`.
